p2p/protocols/protocol.go

```diff
@@ -112,6 +112,14 @@ func errorf(code int, format string, params ...interface{}) *Error {
 	}
 }
 
+// TestMsgPauser can be used to pause run execution
+// IMPORTANT: should be used only for tests
+type TestMsgPauser interface {
```

Comment:
Event thou it should be used only in tests, I think that a comment is good enough and to have the interface named only Pauser or to have consistency on other names, MsgPauser.